editors/vscode/src/client.ts

```diff
@@ -506,6 +518,14 @@ export interface Env {
 }
 
 export interface Goal {
+    typeofgoal : String // type of goal
```

Comment:
Isn't String a too big type? There are only two possible values: UnifGoal and TypGoal. Is it possible to have enumerated types or just a boolean, or is it useful to have a String?

Reply:
typeofgoal doesn't seem to be used anywhere. Why?

Reply:
I see. UnifGoal and TypGoal add fields to Goal. So both UnifGoal and TypGoal have the field typeofgoal. Right?

Reply:
I kept `typeofgoal` as string, because it was used to parse the incoming JSON

Reply:
yes both of them will have the `typeofgoal` field

Reply:
I see. Could you please add in a comment that the values are defined in lsp_base.ml?